pkgs/by-name/ti/ticktick/package.nix

```diff
@@ -72,7 +80,10 @@ stdenv.mkDerivation (finalAttrs: {
     homepage = "https://ticktick.com/home/";
     license = licenses.unfree;
     maintainers = with maintainers; [ hbjydev ];
-    platforms = [ "x86_64-linux" ];
+    platforms = [
+      "x86_64-linux"
+      "aarch64-linux"
+    ];
```

Comment:
Has this been `nixpkgs-fmt`'ed? I think these should be collapsed onto one line.

Reply:
I think its formatted. And its passing formatting check.